Cargo.toml

```diff
@@ -318,7 +318,7 @@ num-bigint = "0.4.4"
 num-prime = "0.4.4"
 num-traits = "0.2.19"
 number_prefix = "0.4"
-onig = { version = "~6.4", default-features = false }
+onig = { version = "~6.5.0", default-features = false }
```

Comment:
Not a huge deal since you bumped onig-sys as well, but you might want to pick 6.5.1 directly.

Reply:
Good point, it's fixed now. Thanks!